src/cennznut/v0/mod.rs

```diff
@@ -31,6 +31,7 @@ pub const MAX_MODULES: usize = 256;
 pub const MAX_METHODS: usize = 128;
 pub const MAX_CONTRACTS: usize = 255;
 pub const VERSION_BYTES: [u8; 2] = [0, 0];
+pub const MAX_CENNZNUT_BYTES: usize = 0xffff;
```

Comment:
plain old base 10 would be easier 😉

Reply:
I gotta disagree with you on this one; the max-bytes are based off of a u16, so representing it as the max u16 value is clearer IMO 😉

Reply:
Actually, let's compromise, I'll use u16::MAX.into()